hack/verify-api-reference-docs.sh

```diff
@@ -29,14 +29,14 @@ API_REFERENCE_DOCS_ROOT="${KUBE_ROOT}/docs/api-reference"
 OUTPUT_DIR="${KUBE_ROOT}/_tmp/api-reference"
 mkdir -p ${OUTPUT_DIR}
 TMP_ROOT="${KUBE_ROOT}/_tmp"
+trap "rm -rf ${TMP_ROOT}" EXIT SIGINT
```

Comment:
Would it make sense to `TMP_ROOT=$(mktemp -d)` instead? It would give us better guarantee that it won't collide again (process could die for lots of different reasons beside EXIT and SIGINT)

mktemp(1) is not POSIX, but I understand that it's not a problem :-)